### PR TITLE
Go live with new cycle

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
New recruitment cycle

### Changes proposed in this pull request
- Remove landing page RE: cycle closing
- Remove apply CTA until 8 Oct
- Remove financial incentives
- Add accessibility statement

### Guidance to review
# After - course page
localhost_5000_course_26M_37VC(Laptop with MDPI screen)

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Replace apply information on course pages](https://trello.com/c/SBiO1aNy/2194-replace-apply-information-on-course-pages)

**Comment by dankmitchell** [comment]
[Replace financial incentives information](https://trello.com/c/tgqOurDi/2193-replace-financial-incentives-information)

**Comment by dankmitchell** [comment]
[Add FIND accessibility statement](https://trello.com/c/J1BS2Bau/2212-add-find-accessibility-statement)

**Comment by dankmitchell** [comment]
Ready for review @fofr https://search-and-compare-ui-pr-427.herokuapp.com/

File: shared/Views/Shared/Components/CourseDetails/Default.cshtml [hunk 1]
```diff
@@ -140,16 +126,12 @@
...
-          else
-          {
-            @await Html.PartialAsync("_Fees.cshtml")
-          }
-
-          @await Html.PartialAsync("FinancialSupport/_FinancialSupport.cshtml")
+          <div class="govuk-!-margin-bottom-8">
+            <h2 class="govuk-heading-l" id="section-financial-support">Financial support</h2>
+            <div class="govuk-inset-text">
+              <p class="govuk-body">The government will soon announce the financial incentives for upcoming courses in 2020/21</p>
```
Comment (fofr) [inline shared/Views/Shared/Components/CourseDetails/Default.cshtml]: Bursaries, scholarships and financial support for 2020/21 will be announced soon.
```suggestion
              <p class="govuk-body">Bursaries, scholarships and financial support for 2020/21 will be announced soon.</p>
```

File: src/Views/Legal/Accessibility.cshtml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+@{ ViewBag.Title = $"Find teacher training courses"; }
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
+  <div class="govuk-width-container">
+    <h1 class="govuk-heading-xl">Accessibility statement for Find teacher training courses</h1>
```
Comment (fofr) [inline src/Views/Legal/Accessibility.cshtml]: All this content needs to be in a 2/3rds container to stop line length being too long.

**Pull request merged**